BETA release post. (#2108)
electric-sql/electric: website/blog/posts/2024-12-09-electric-beta-release.md

---
title: "Electric BETA release"
description: >-
  The Electric sync engine is now in BETA.
  If you haven't checked out Electric recently,
  it's a great time to take another look.
excerpt: >-

>   With version 1.0.0-beta, the Electric sync engine is now in BETA!

Reviewer: ```suggestion
  With version 1.0.0-beta.0, the Electric sync engine is now in BETA!
```

  import LogoStripSm from '/static/img/blog/electric-beta-release/logo-strip.sm.svg'
  import LogoStripXs from '/static/img/blog/electric-beta-release/logo-strip.xs.svg'
  import LogoStripXxs from '/static/img/blog/electric-beta-release/logo-strip.xxs.svg'
  import LinearLiteScreenshot from '/static/img/blog/electric-beta-release/linearlite-screenshot.png'
  import ScalabilityChart from '../../src/components/ScalabilityChart.vue'

  import { onMounted } from 'vue'

  import { data as demosData } from '../../data/demos.data.ts'
  const { demos } = demosData

  const notesDemo = demos.find(x => x.link === '/demos/notes')
  const pixelArtDemo = demos.find(x => x.link === '/demos/pixel-art')

  onMounted(async () => {
    if (typeof window !== 'undefined' && document.querySelector) {
      let links = document.querySelectorAll('.cloud-cta a.VPButton.brand')

      links.forEach((link) => {
        if (link.querySelector('span.vpi-electric-icon')) {
          return
        }

        const icon = document.createElement('span')
        icon.classList.add('vpi-electric-icon')

        link.prepend(icon)
      })
    }
  })
</script>

> With version [`1.0.0-beta`](https://github.com/electric-sql/electric/blob/main/packages/sync-service/CHANGELOG.md) the Electric sync engine is now in BETA!

Reviewer: ```suggestion
With version [`1.0.0-beta.0`](https://github.com/electric-sql/electric/blob/main/packages/sync-service/CHANGELOG.md) the Electric sync engine is now in BETA!
```

> *&mdash; [Vache Asatryan](https://am.linkedin.com/in/vacheasatryan), Co-founder &amp; CTO, [Doorboost](https://doorboost.com)*

### Scalable

So many real-time sync systems demo well but break under real load.

Electric has been [engineered from the ground up](/docs/api/http) to handle high-throughput workloads, like [Trigger.dev](https://trigger.dev/launchweek/0/realtime), with low latency and flat resource use.

The chart below is from our [cloud benchmark testing](/docs/reference/benchmarks) showing Electric streaming real-time data to **millions of concurrent users** from a **single commodity Postgres** with linear CPU and flat memory use:

<figure>
  <ScalabilityChart />
</figure>

You can also see how large-scale apps built with Electric feel to use with our updated [ Linearlite](/demos/linearlite) demo. This is a [Linear](https://linear.app) clone that loads 100,000k issues and their comments through Electric into PGlite. It loads fast, it feels instant and it's fully interactive:

<figure>
  <a href="https://linearlite.examples.electric-sql.com" target="_blank">
    <img :src="LinearLiteScreenshot" />
  </a>
  <figcaption>

>     Screenshot of Linearlite. Click on it to

Reviewer: ```suggestion
    Screenshot of Linearlite.
```

### With your existing API

Because Electric syncs [over HTTP](/docs/api/http), you can use it together [with your existing API](/blog/2024/11/21/local-first-with-your-existing-api).

This allows you to handle concerns like [auth](/docs/guides/auth) and [writes](/docs/guides/writes) with your existing code and web service integrations. You don't need to codify your auth logic into database rules. You don't need to replace your API endpoints and middleware stack.

## Take another look

With this BETA release, Electric is stable and ready for prime time use. If you haven't checked it out recently, it's a great time to take another look.

<div class="actions cta-actions page-footer-actions left">
  <div class="action">
    <VPButton
        href="/docs/quickstart"
        text="Quickstart"
        theme="brand"
    />
  </div>
  <div class="action">
    <VPButton
        href="/docs/intro"
        text="Documentation"
        theme="alt"
    />
  </div>
</div>

> ### Signup for cloud

Reviewer: ```suggestion
### Signup for early access to Electric Cloud
```

> We're also working hard building our [Cloud product](/product/cloud), which provides managed Electric hosting so you don't need to [host Electric yourself](/docs/guides/deployment).

Reviewer: ```suggestion
We're also building a [Electric Cloud](/product/cloud), which provides managed Electric hosting.
```

> If you're interested in using Electric Cloud, you can sign up for early access now:

Reviewer: ```suggestion
We'll be opening access soon in a private beta so if interested, you can sign up for early access now:
```
